src/main/java/de/lh/tool/domain/dto/NeedDto.java

```diff
@@ -0,0 +1,19 @@
+package de.lh.tool.domain.dto;
+
+import de.lh.tool.domain.model.HelperType;
+import lombok.Data;
+
+@Data
+public class NeedDto {
+
+	private Long id;
+
+	private Long projectId;
+
+	private Long date;
```

Comment:
date als Long?

Reply:
Finde ich persönlich am Einfachsten zum Austausch (in Millisekunden), weil es da eigentlich überall einen Konstruktor gibt. Willst du lieber Timestamp-Strings oder yyyy-mm-dd?

Reply:
Hätte jetzt an einen nativen Date-Typ gedacht, also so ungefähr:
Der Zeitpunkt interessiert uns doch eigentlich gar nicht (oder?). Wir brauchen ja nur das Datum.

Reply:
Ja, brauchen wir nicht. Kann ich gerne machen, ich glaube Spring macht da dann im JSON als default auch ein Long draus

Reply:
OK, was im JSON draus wird weiß ich nicht.
Dritte Alternative wäre natürlich, wenn wir uns ein eigenes primitives Date-Objekt basteln (das dürfte ja dann in seine Member serialisiert werden.

Reply:
Auf jeden Fall, wäre irgendeine Abstraktion über den nackten Long ganz schön.